### DISCUSSION TITLE
mmcv.runner cannot find the reference

### DISCUSSION THREAD

**Discussion by sidiangongyuan** [open-mmlab/mmcv#2932] [Q&A]
mmcv 2.0.0
mmdet 3.1.0
mmdet3d 1.2.0
cuda version 11.6
python 3.8

I have the same question on :
 from mmcv.cnn import xavier_init, constant_init   
from mmcv.runner import force_fp32, auto_fp16
from mmcv.runner.base_module import BaseModule, ModuleList, Sequential

**Comment by XU-GITHUB-curry** [comment]
did you solve it, please? I met the same problem

**Reply by sidiangongyuan** [reply]
> did you solve it, please? I met the same problem

Yes, you need to downgrade mmcv version or I find from mmcv.cnn import xavier_init, constant_init in : 
from mmengine.model import xavier_init, bias_init_with_prob

here is my new version:
mmcv-full                 1.6.0                    pypi_0    pypi
mmdet                     2.24.0                   pypi_0    pypi
mmdet3d                   1.0.0rc5                  dev_0    <develop>
